src/Effects.js

```diff
@@ -1445,6 +1449,67 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.emojis = {
+      emojiList : [],
+      emojiTypes : [],
+
+      init : function () {
+        this.imageLovestruck = p5.createGraphics(100, 100);
+        this.imageLovestruck.scale(3);
+        drawLovestruck(this.imageLovestruck.drawingContext);
+        this.emojiTypes.push(this.imageLovestruck);
+
+        this.imageSmiley = p5.createGraphics(100, 100);
+        this.imageSmiley.scale(3);
+        drawSmiley(this.imageSmiley.drawingContext);
+        this.emojiTypes.push(this.imageSmiley);
+
+        this.imageStarstruck = p5.createGraphics(100, 100);
+        this.imageStarstruck.scale(3);
+        drawStarstruck(this.imageStarstruck.drawingContext);
+        this.emojiTypes.push(this.imageStarstruck);
+
+        this.imageTickled = p5.createGraphics(100, 100);
+        this.imageTickled.scale(3);
+        drawTickled(this.imageTickled.drawingContext);
+        this.emojiTypes.push(this.imageTickled);
+
+        this.imageWink = p5.createGraphics(100, 100);
+        this.imageWink.scale(3);
+        drawWink(this.imageWink.drawingContext);
+        this.emojiTypes.push(this.imageWink);
+      },
+
+      draw : function () {
+        if (p5.frameCount % 10 === 0) {
+          this.emojiList.push(new this.emoji(this.emojiTypes, this.emojiList));
+        }
+        for (let aEmoji of this.emojiList) {
+          aEmoji.update();
+          aEmoji.display();
+        }
+      },
+
+      emoji : function (emojiTypes, emojiList) {
+        this.x = randomNumber(0, 350);
+        this.y = -50;
+        this.size = randomNumber(25, 50);
+        let emojiImage = emojiTypes[randomNumber(0, 4)];
+
+        this.update = function () {
+          this.y += p5.pow(this.size, 0.25);
+          if (this.y > p5.height * 1.2) {
```

Comment:
Similar to explaining the 0.25, can we add a short comment explaining the 1.2 value?